datajunction-server/datajunction_server/internal/nodes.py

```diff
@@ -1224,7 +1224,9 @@ async def create_new_revision_from_existing(
         type=old_revision.type,
         columns=[
             Column(
-                name=column_data.name,
+                name=column_data.name.lower()
+                if node.type != NodeType.METRIC
+                else column_data.name,
```

Comment:
I'm curious why we wouldn't do it for metric nodes too. Is it just because we don't have to? Is there a case to be made for consistency?

Reply:
We introduce the `DOT` separator ourselves for metrics, which already has the upper case so it wouldn't make sense to normalize for those as well.